cli/dcoscli/job/main.py

```diff
@@ -796,7 +796,7 @@ def _get_api_url(path):
 
 def _get_timeout():
     """
-    :returns: timout value for API calls
+    :returns: timeout value for API calls
```

Comment:
Unrelated fix.

Reply:
Good catch, feel free to update the PR to only keep this commit and I'll merge.